builtin/k8s/platform.go

```diff
@@ -376,7 +394,7 @@ func (p *Platform) Deploy(
 
 		if time.Since(lastStatus) > 10*time.Second {
 			step.Update(fmt.Sprintf(
-				"Waiting on deployment to become available: %d/%d/%d",
+				"Waiting on deployment to become available: %d/%d/%d\n",
```

Comment:
I'm actually wondering if adding this line break isn't masking a problem. Without it, log output is interwoven.